pages/index.js

```diff
@@ -43,15 +47,58 @@ export default function Home() {
         <div>
           <label>Aktivitäten</label>
           <select name="activities">
-            <option value="">Bitte auswählen</option>
+            <option value="auswahl">Bitte auswählen</option>
             <option value="gassigehen">Gassi gehen</option>
             <option value="sport">Sport</option>
             <option value="otherthings">Sonstiges</option>
           </select>
         </div>
+        <div>
+          <div>
+            <h4>Altersgruppe</h4>
+            <div>
+              <div>
+                <label htmlFor="welpen">Welpen</label>
+                <input
+                  type="checkbox"
+                  name="ageGroup"
+                  id="welpen"
+                  data-displayname="Welpen"
+                />
+              </div>
+              <div>
+                <label htmlFor="junghunde">Junghunde</label>
+                <input
+                  type="checkbox"
+                  name="ageGroup"
+                  id="junghunde"
+                  data-displayname="Junghunde"
+                />
+              </div>
+              <div>
+                <label htmlFor="erwachsen">Erwachsen</label>
+                <input
+                  type="checkbox"
+                  name="ageGroup"
+                  id="erwachsen"
+                  data-displayname="Erwachsen"
+                />
+              </div>
+              <div>
+                <label htmlFor="senior">Senior</label>
+                <input
+                  type="checkbox"
+                  name="ageGroup"
+                  id="senior"
+                  data-displayname="Senior"
+                />
+              </div>
+            </div>
+          </div>
+        </div>
 
         <div>
-          <button type="submit">Erstellen</button>
+          <StyledButton>ADD</StyledButton>
         </div>
       </form>
```

Comment:
- Consider exporting <form> and <ul> out of the index.js like you did with <Title> otherwise you lose overview.